app/views/browser/_search_control.html.erb

```diff
@@ -56,32 +64,9 @@
 
   <div layout="row" layout-align="center center">
     <md-button ng-class="views.search.selected ? 'md-raised md-primary' : 'md-raised'" 
-               ng-click="search(0); select_view('search')"
+               ng-click="search(0); select_view('search'); item_search()"
```

Comment:
I think that `item_search` needs to move into `search`, so this shouldn't be here.